presidio-image-anonymizer/presidio_image_anonymizer/image_analyzer_engine.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+from presidio_image_anonymizer.image_recognizer_result import ImageRecognizerResult
+from presidio_image_anonymizer.ocr import OCR
+from presidio_analyzer import RecognizerResult
+from presidio_analyzer import AnalyzerEngine
+
+
+class ImageAnalyzerEngine:
+    """ImageAnalyzerEngine class."""
+
+    def analyze(self, image: object) -> List[ImageRecognizerResult]:

```

Comment:
Should we expose **kwargs to be used to customize the text analyzer call?

Reply:
done